lib/salesforce/config.rb

```diff
@@ -6,12 +6,12 @@ class Config
     include Blockenspiel::DSL
     include Blockenspiel::DSLSetupMethods
 
-    dsl_attr_accessor :session_id, :server_instance, :user_id, :soap_endpoint_url
```

Comment:
If we plan to bump a minor or patch, you cannot introduce change that is not backward compatible

Reply:
Ah yeah, originally planned on this being a major bump, but forgot about that breaking change when trying to do a minor one

Reply:
I think it would still make sense to do this eventually, but for now I will come up with something non-breaking

Reply:
I revised the approach here, let me know what you think